publication-rest/src/main/java/no/unit/nva/publication/update/PublicationRequest.java

```diff
@@ -0,0 +1,15 @@
+package no.unit.nva.publication.update;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonSubTypes({
+    @JsonSubTypes.Type(UpdatePublicationRequest.class),
+    @JsonSubTypes.Type(UnpublishPublicationRequest.class),
+    @JsonSubTypes.Type(DeletePublicationRequest.class)
+})
+@SuppressWarnings({"PMD.AbstractClassWithoutAbstractMethod", "PMD.AbstractClassWithoutAnyMethod"})
```

Comment:
Probably interface, seems like marker pattern rather than inheritance of some common thing…the child classes don't have much in common at a semantic level aside from "does something to a publication", and don't share much else.

Reply:
my reasoning was the the JsonSubTypes stuff did not carry along with the interface, but looks like it works now, so well well. I`ll fix it